Pipfile

```diff
@@ -4,11 +4,17 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-alembic = "==0.9.8"
-"psycopg2" = "==2.7.4"
-SQLAlchemy = "==1.2.5"
+alembic = "*"
+"psycopg2" = "*"
+SQLAlchemy = "*"
+flask = "*"
+flask-sqlalchemy = "*"
+flask-migrate = "*"
+"e1839a8" = {path = ".", editable = true}
```

Comment:
What is this?

Reply:
this is the local, editable install of our flask app (`pip install -e .`)
it's what allows specifying `FLASK_APP=api` instead of providing the path to the project.